metecho/api/models.py

```diff
@@ -440,6 +462,65 @@ def get_collaborator(self, gh_uid: str) -> Optional[Dict[str, object]]:
             return None
 
 
+class GitHubOrganization(HashIdMixin, PushMixin, TimestampsMixin):
+    """
+    Used to keep a list of known GH orgs and their members.
+    """
+
+    name = StringField()
+    login = StringField(unique=True, help_text="Organization's 'username' on GitHub")
+    avatar_url = models.URLField(blank=True)
+    # User data is shaped like this:
+    #   {
+    #     "id": str,
+    #     "login": str,
+    #     "avatar_url": str,
+    #   }
+    members = models.JSONField(blank=True, default=list)
```

Comment:
Why do we need the members? It could be quite large. If it's to determine which orgs the user can create a repository in, maybe there is a way to query it just for one user at the same time that we refresh the user's repositories? That way we don't have to store anything except for the users who have logged into Metecho.

Reply:
This is used to render the list of users in the "Add Project Collaborators" step of the wizard. Should we pull from the Metecho users instead? 

![image](https://user-images.githubusercontent.com/2391102/129226257-e9ccbae7-cdcc-4057-9861-e8533da9dea0.png)

Reply:
Ah ok, we can discuss at checkin

Reply:
It looks like members can choose to [show/hide their membership](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership) from the public, but other [organization members will always be able to see them](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/permission-levels-for-an-organization#permission-levels-for-an-organization).

I think the best approach is to follow GitHub. We can create a new endpoint `/api/organizations/:id/members/` that connects to GH as the current user and queries for the members. The frontend will need to display a spinner while we wait for the response (shouldn't be an issue since we do that in other places already).

Reply:
Even aside from the need to make a user-specific query, it makes sense to me to do this on demand rather than when the organization model is set up anyway, since it will be an infrequent operation.

Reply:
Refactored in 2014cbea15886574c4f316e8cc371464668b600c